content/W2015/W15-event-research-showcase.md

```diff
@@ -17,8 +17,8 @@ connect with them and get to know more in a mixer setting.
 
 ## Event Details ##
 
-+ **Who:** [Cecylia Bocovich](https://cs.uwaterloo.ca/~cbocovic/), [Marianna
-  Rapoport](https://cs.uwaterloo.ca/~mrapopor/index.html), more TBA
++ **Who:** [Cecylia Bocovich](https://cs.uwaterloo.ca/~cbocovic/), [Erinn Atwater]
+(https://cs.uwaterloo.ca/~aatwater/),[Carolyn Lamb], [Laleh Ghoraie] 
```

Comment:
This should have been tested prior to a merge: the `[]()` syntax is specifically for creating links so without the parens, those []'s will render.